### ISSUE TITLE
Issue with '

### ISSUE THREAD

**Issue by szentigrad3** [Artentus/ModMyFactory#11]
when i add the 5dim's mods to the modpacks it doesn't seem to like the ' in 5dim's and if i check in the modpacks.json file it shows as 

```
    {
      "Name": "5dim",
      "FactorioVersion": "0.14"
    },
```

11 times and when i re-open ModMyFactory it shows only the 1st mod 11 times over
Also when starting the modpacks from external links i get: Factorio Version 0.14.14 is not available, Check your installed factorio versions.

pics here: https://s10.postimg.org/ca5xty6yh/modmyfactory.png

**Comment by szentigrad3** [comment]
also i try and launch the game with the modpack i want it starts the game only with the base mod

**Comment by Artentus** [comment]
Don't know what's going on there to be honest. I will get 5dim's myself later today and test.

**Comment by Artentus** [comment]
It actually was not ' but _ that caused this bug.
Fixed as of version 1.2.0.
